apps/server/lib/lexical/server/state.ex

```diff
@@ -234,6 +234,7 @@ defmodule Lexical.Server.State do
         definition_provider: true,
         document_formatting_provider: true,
         hover_provider: true,
+        references_provider: true,
```

Comment:
I think the `FindReferences` handler is still commented out, right? Is this line something from testing that got left in on accident?

Reply:
ah yes. I'll remove.